src/main/java/org/jenkinsci/test/acceptance/plugins/mission_control/MissionControlView.java

```diff
@@ -0,0 +1,285 @@
+     *
+     * @param state The new state of the hidden build queue option.
+     */
+    public void setHideBuildQueue(boolean state) {
+        ensureConfigPage();
+        hideBuildQueue.check(state);
+    }
+
+    /**
+     * Sets whether or not the nodes shall be hidden.
+     * Calls {@link ConfigurablePageObject#ensureConfigPage()} to ensure that the configuration page of the view is active.
+     *
+     * @param state The new state of the hidden nodes option.
+     */
+    public void setHideNodes(boolean state) {
+        ensureConfigPage();
+        hideNodes.check(state);
+    }
+
+    /**
+     * Retrieves all possible options of buttons sizes.
+     * Calls {@link ConfigurablePageObject#ensureConfigPage()} to ensure that the configuration page of the view is active.
+     *
+     * @return List of valid button sizes.
+     */
+    public List<String> getStatusBtnSizes() {
+        ensureConfigPage();
+        List<String> options = new LinkedList<>();
+        for (WebElement e : find(By.xpath("//select[@path='/statusButtonSize']")).findElements(By.tagName("option"))) {
+            options.add(e.getText());
+        }
+        return options;
+    }
+
+    /**
+     * Sets the size of the status buttons. {@link MissionControlView#getStatusBtnSizes()} returns possible size options.
+     * Calls {@link ConfigurablePageObject#ensureConfigPage()} to ensure that the configuration page of the view is active.
+     *
+     * @param sizeOption The new size of the status button.
+     */
+    public void setStatusBtnSize(String sizeOption) {
+        ensureConfigPage();
+        statusBtnSize.select(sizeOption);
+    }
+
+    /**
+     * Retrieves all possible options of height layout ratios.
+     * Calls {@link ConfigurablePageObject#ensureConfigPage()} to ensure that the configuration page of the view is active.
+     *
+     * @return List of valid height layout ratios.
+     */
+    public List<String> getHeightRatios() {
+        ensureConfigPage();
+        List<String> options = new LinkedList<>();
+        for (WebElement e : find(By.xpath("//select[@path='/layoutHeightRatio']")).findElements(By.tagName("option"))) {
+            options.add(e.getText());
+        }
+        return options;
+    }
+
+    /**
+     * Sets the layout height ratio. {@link MissionControlView#getHeightRatios()} returns possible ratio options.
+     * Calls {@link ConfigurablePageObject#ensureConfigPage()} to ensure that the configuration page of the view is active.
+     *
+     * @param ratioOption The new ratio of the layout.
+     */
+    public void setHeightRatio(String ratioOption) {
+        ensureConfigPage();
+        heightRatio.set(ratioOption);
+    }
+
+    /**
+     * Reloads the configuration of jenkins from disk.
+     * Necessary to display the build history if a new {@link MissionControlView} is created.
+     * (Inefficient) alternative: {@code jenkins.restart()}
```

Comment:
It is required (see: [https://wiki.jenkins-ci.org/display/JENKINS/Mission+Control+Plugin](url) at "Setting Up"). I don't know how mission control works internally to display the build history, but it seems the plugin developers are aware of this behavior.